doc/source/development/contributing_environment.rst

```diff
@@ -204,9 +244,22 @@ At this point you should be able to import pandas from your locally built versio
    >>> print(pandas.__version__)  # note: the exact output may differ
    2.0.0.dev0+880.g2b9e661fbb.dirty
 
-This will create the new environment, and not touch any of your existing environments,
-nor any existing Python installation.
-
-.. note::
-   You will need to repeat this step each time the C extensions change, for example
-   if you modified any file in ``pandas/_libs`` or if you did a fetch and merge from ``upstream/main``.
+When building pandas with meson, importing pandas will automatically trigger a rebuild, even when C/Cython files are modified. 
+By default, no output will be produced by this rebuild (the import will just take longer). If you would like to see meson's 
+output when importing pandas, you can set the environment variable ``MESONPY_EDTIABLE_VERBOSE``. For example, this would be::
+
+   # On Linux/macOS
+   MESONPY_EDITABLE_VERBOSE=1 python
+   
+   # Windows
+   set MESONPY_EDITABLE_VERBOSE=1 # Only need to set this once per session
+   python
+
+If you would like to see this verbose output every time, you can set the ``editable-verbose`` config setting to ``true`` like so::
+   
+   python -m pip install -ve . --config-settings editable-verbose=true
```

Comment:
This does not work with meson 1.0.0

```
meson-python: error: Unknown configuration entry "editable-verbose"
```

Reply:
Hm, what version of meson-python are you using?
You need to install from main in order for editable installs to work.
Try reinstalling it.
e.g. ``pip uninstall meson-python && pip install git+https://github.com/mesonbuild/meson-python.git@main``

Reply:
Yeah I already installed from meson main. No change when reinstalling

Reply:
```
meson==1.0.0
meson-python @ git+https://github.com/mesonbuild/meson-python.git@8a13f8d199e4961fe2b0aeff01d231f510b6c59b
```

output from pip freeze

Reply:
Is this still outstanding?

Reply:
Sorry for not getting back on this. This works for me on macOS.
Perhaps it was a bug in meson-python < 0.13.1

Reply:
This works with 0.13.1 indeed, it can be resolved now. Editable support was added in 0.13.0 - the whole implementation changed after the Jan 4 comments.